index.html

```diff
@@ -1,19 +1,18 @@
 <!doctype html>
-<html lang="{{PAGE_LANGUAGE}}">
```

Comment:
Doesn't Vite also have the ability to use variables in html?

https://vitejs.dev/guide/env-and-mode.html#html-env-replacement

Reply:
Yes, this was only done for rapid prototyping.